### PR TITLE
Command update checks

### PR DESCRIPTION
**Pull request by Therzok**

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Components.Commands/CommandManager.cs [hunk 1]
```diff
@@ -2245,24 +2245,24 @@ public void Run (object cmdTarget, CommandInfo info)
 			if (customHandlerChain != null) {
 				info.UpdateHandlerData = Method;
 
-				DateTime t = DateTime.Now;
```
Comment (slluis) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Components.Commands/CommandManager.cs]: We should actually use Stopwatch

Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Components.Commands/CommandManager.cs]: I'm probably pedantic here, but DateTime.UtcNow is faster than Stopwatch based on my profiling, and we don't need nanosecond resolution. I didn't see it necessary to switch to stopwatch for that.

Comment (slluis) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Components.Commands/CommandManager.cs]: Is it so faster so that it is worth losing accuracy?

**Comment by Therzok** [comment]
Manually merged to master with `Stopwatch` instead of DateTime.UtcNow.

**Pull request closed without merging**